pages/docs/community/styleguide/content-buckets.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Content Buckets
+Description: This style guide explains how and why we use content buckets in documentation.
+weight: 30
+---
+
+# Content Buckets
+To structure our documentation, we adopt a systematic approach from the Diátaxis framework. We ensure our docs are presented in a manner that is easy to understand by organizing and classifying them into content buckets.
+
+![Diátaxis framework](https://www.asyncapi.com/img/posts/changes-coming-docs/diataxis.webp)
+
+> Documentation structure based on the [Diátaxis framework](https://diataxis.fr/).
+
+At AsyncAPI our docs are classified into the following content buckets:
+
+- **Concepts** - This section is where we define various features and capabilities AsyncAPI offers. All docs pages under this bucket need to be accurate, easy to understand and accompanied by an engineering diagram. We utilize [Mermaid.js](https://mermaid.js.org/) syntax to create diagrams and visualizations.
```

Comment:
```suggestion
- **Concepts** - This section defines various features and capabilities AsyncAPI offers. 
```

We use mermaid diagrams anywhere in the docs, and there is no specific rule that only concept pages must have them.